app/_kong_plugins/cors/examples/basic-configuration.yaml

```diff
@@ -0,0 +1,28 @@
+title: Configure the plugin on a Service
```

Comment:
The targets are set by the platform based on the plugin's support, so there's no need to specify it here or [here](https://github.com/Kong/developer.konghq.com/pull/173/files#diff-5e3f90dd52aa9a4d0e8c32238de99cd3a50f4ab2fb414e9a3062d14cac230931R10).

I guess the goal of this is to allow requests from a specific domain (http://mockbin.com) and only two methods.
We could drop the headers and go with the default ones to make it simpler and rename the file, title and description to something like `Allow requests from a specific domain`